vaadin-core-internal/pom.xml

```diff
@@ -209,6 +209,11 @@
             <artifactId>vaadin-menu-bar-flow</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.vaadin.hilla</groupId>
+            <artifactId>hilla</artifactId>
```

Comment:
Also I think we have to make changes in `hilla` artifact, so here are some possible actions:

1. `pom.xml` can be retained as is in `hilla`.
2. `@NpmPackage` annotations should be removed from `hilla`.
3. `hilla-react` can be just removed from Hilla as it has the same pom.xml as `hilla`.
4. `hilla` then is added to Platform.
5. Flow adds these three react packages (Auto Crud etc.) if `reactRouterEnabled=true` otherwise it adds one lit form package (in `NodeUpdater` class).

Reply:
> Flow adds these three react packages (Auto Crud etc.) if reactRouterEnabled=true otherwise it adds one lit form package (in NodeUpdater class).

I would vote against this; or add "if hilla is on the class path" - otherwise flow only apps gets react packages thanks to reactRouterEnabled=true as default

Reply:
> or add "if hilla is on the class path"

Yes, agree, that's a good point.

I proposed to remove npm package annotations and add packages imperatively in NodeUpdater, so that to relief developers from excluding artifacts by themselves.

Reply:
Updated the wording for 5.